### ISSUE TITLE
HELLO, I Want to get a driver of HPE CRAY EX235A.

### ISSUE THREAD

**Issue by yanbosmu** [Cray-HPE/docs-csm#4582]
hello, EVERYONE!
I got a AMD MI250X GPU card recently. Obviously it was a part from HPE cray EX235a system.
But I got problem in using it now in ubuntu. Probably no driver suitable in ubuntu?
So, do you have drivers or BIOS that can make my card alive? It is writtern D65201-OB Rev13 on GPU.
THANK YOU SO MUCH!

**Comment by rustydb** [comment]
@yanbosmu apologies on the lengthy delay, we seldom receive issues via GitHub and apparently we neglect to check them.
Was your issue ever answered by other means or do you still require assistance?

**Comment by yanbosmu** [comment]
Yes, I still need your help！ I am still in trouble getting this card running.
